src/Libraries/CoreNodes/DSCoreNodesImages.resx

```diff
@@ -5119,44 +5119,6 @@
         gOTfKGA1R7m+SSE8APXA8bsfLqezAAAAAElFTkSuQmCC
 </value>
   </data>
-  <data name="DSCore.IO.File.ReadText.Large" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
```

Comment:
@sharadkjaiswal, we are collating the File Class from DSCore to DSCore.IO, there are two methods with same name called ReadText. To allow overloading (i.e. pointing to the correct image), we need to change the filename. See anna's comments for further detail.